lib/active_record/connection_adapters/redshift_adapter.rb

```diff
@@ -190,7 +190,7 @@ def initialize(connection, logger, connection_parameters, config)
       end
 
       # Clears the prepared statements cache.
-      def clear_cache!
+      def clear_cache!(new_connection: true)
```

Comment:
For what it's worth, `false` is the default value of this parameter in Rails, so I'd rather keep it that way.

```suggestion
      def clear_cache!(new_connection: false)
```